datahub-web-react/src/app/ingest/source/IngestedAssets.tsx

```diff
@@ -0,0 +1,174 @@
+        entityTypeFacets.aggregations.forEach((agg) =>
+            finalCounts.push({
+                count: agg.count,
+                displayName: pluralize(agg.count, capitalizeFirstLetterOnly(agg.value) || ''),
+            }),
+        );
+    }
+
+    return finalCounts;
+};
+
+export default function IngestedAssets({ urn, id }: Props) {
+    // First thing to do is to search for all assets with the id as the run id!
+    console.log(urn);
+
+    const [showAssetSearch, setShowAssetSearch] = useState(false);
+
+    // Execute search
+    const { data, loading, error } = useGetSearchResultsForMultipleQuery({
+        variables: {
+            input: {
+                query: '*',
+                start: 0,
+                count: 1,
+                filters: [
+                    {
+                        field: 'runId',
+                        value: id,
+                    },
+                ],
+            },
+        },
+    });
+
+    // Parse filter values to get results.
+    const facets = data?.searchAcrossEntities?.facets;
+    const entityTypeFacets = facets?.filter((facet) => facet.field === 'entity')[0];
+
+    const hasSubTypeFacet = (facets?.findIndex((facet) => facet.field === 'typeNames') || -1) >= 0;
+    const subTypeFacets = (hasSubTypeFacet && facets?.filter((facet) => facet.field === 'typeNames')[0]) || undefined;
+    const countsByEntityType =
+        (entityTypeFacets && extractEntityTypeCountsFromFacets(entityTypeFacets, subTypeFacets)) || [];
+    const total = data?.searchAcrossEntities?.total || 0;
+
+    return (
+        <>
+            {error && <Message type="error" content="" />}
+            <>
+                <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+                    <div>
+                        <Typography.Title level={5}>Ingested Assets</Typography.Title>
+                        {(loading && <Typography.Text type="secondary">Loading...</Typography.Text>) || (
+                            <>
+                                {(total > 0 && (
+                                    <Typography.Paragraph type="secondary">
+                                        The following asset types were ingested during this run.
+                                    </Typography.Paragraph>
+                                )) || (
+                                    <Typography.Text style={{ fontSize: 12 }}>No assets were ingested.</Typography.Text>
+                                )}
+                            </>
+                        )}
+                    </div>
+                    {!loading && (
+                        <div
+                            style={{
+                                display: 'flex',
+                                flexDirection: 'column',
+                                justifyContent: 'right',
+                                alignItems: 'end',
+                            }}
+                        >
```

Comment:
could use styled components here (and elsewhere in this return method) to clean things up a bit

Reply:
ah but just remembering you already mentioned you need to address this

Reply:
Doing this pass now!